src/main/resources/schemas/nhinc/common/configAdmin.xsd

```diff
@@ -28,73 +28,81 @@
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListKeyStoresRequestMessage" type="tns:ListKeyStoresRequestMessageType" />
+    
+    <xsd:complexType name="ListKeyStoresResponseMessageType">
+         <xsd:sequence>
+            <xsd:element name="certList" type="tns:ListCertificateType" minOccurs="0" maxOccurs="unbounded" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListKeyStoresResponseMessage" type="tns:ListKeyStoresResponseMessageType"/>
     
     <xsd:complexType name="ListTrustStoresRequestMessageType" >
         <xsd:sequence>
             <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="refreshCertCache" minOccurs="1" maxOccurs="1" default="false" />
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="ListTrustStoresRequestMessage" type="tns:ListTrustStoresRequestMessageType" />
     
     <xsd:complexType name="ListTrustStoresResponseMessageType">
          <xsd:sequence>
-            <xsd:element name="certData" type="xsd:base64Binary" xmime:expectedContentTypes="application/octet-stream" minOccurs="0" maxOccurs="unbounded"/>
+            <xsd:element name="certList" type="tns:ListCertificateType" minOccurs="0" maxOccurs="unbounded" />
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="ListTrustStoresResponseMessage" type="tns:ListTrustStoresResponseMessageType"/>
     
+    <xsd:complexType name="ListCertificateType">
+         <xsd:sequence>
+			<xsd:element name="alias" type="xsd:string" />
+            <xsd:element name="certData" type="xsd:base64Binary" xmime:expectedContentTypes="application/octet-stream" />
+        </xsd:sequence>
+    </xsd:complexType>
+    <xsd:element name="ListCertificate" type="tns:ListCertificateType" />
+    
+    
+    
     <xsd:complexType name="EditCertificateRequestMessageType" >
         <xsd:sequence>
             <xsd:element name="editCertRequest" type="tns:EditCertificateRequestType" minOccurs="1" maxOccurs="1" />
             <xsd:element name="configAssertion" type="tns:ConfigAssertionType" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="isTrustStore" type="xsd:boolean" minOccurs="1" maxOccurs="1" default="true" />
         </xsd:sequence>
     </xsd:complexType>
     <xsd:element name="EditCertificateRequestMessage" type="tns:EditCertificateRequestMessageType" />
     
     <xsd:complexType name="EditCertificateRequestType">
         <xsd:sequence>
-            <xsd:element name="alias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+            <xsd:element name="oldAlias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="newAlias" type="xsd:string" minOccurs="1" maxOccurs="1" />
+			<xsd:element name="certData" type="xsd:base64Binary" xmime:expectedContentTypes="application/octet-stream"  minOccurs="1" maxOccurs="1" />
```

Comment:
Why is certData being passed here?  I thought this was just to update the alias?